service/importer/import.go

```diff
@@ -2,30 +2,30 @@ package importer
 
 import (
 	"github.com/mesg-foundation/core/service"
-
 	yaml "gopkg.in/yaml.v2"
 )
 
-// From imports a service from a source
-func From(source string) (importedService *service.Service, err error) {
+// From imports a service from a source.
+func From(source string) (*service.Service, error) {
 	return fromPath(source)
 }
 
-// fromPath imports a service from a path
-func fromPath(path string) (importedService *service.Service, err error) {
+// fromPath imports a service from a path.
+func fromPath(path string) (*service.Service, error) {
 	isValid, err := IsValid(path)
 	if err != nil {
-		return
+		return nil, err
 	}
 	if isValid == false {
-		err = &ValidationError{}
-		return
+		return nil, &ValidationError{}
 	}
 	data, err := readServiceFile(path)
 	if err != nil {
-		return
+		return nil, err
+	}
+	var importedService service.Service
+	if err := yaml.UnmarshalStrict(data, importedService); err != nil {
```

Comment:
can compact and also i'm not sure this works, in the first version we ad a pointer, in this case we don't and the unmarshal might have some problem to write data

```
return &importedService, yaml.UnmarshalStrict(data, &importedService)
```

Reply:
yes issue with not passing the pointer, fixed it